src/wizard/SetupWizard.php

```diff
@@ -162,9 +162,10 @@ private function generateBaseConfig() : void{
 		$this->message($this->lang->translate(KnownTranslationFactory::gamemode_info()));
 
 		do{
-			$gamemode = (int) $this->getInput($this->lang->translate(KnownTranslationFactory::default_gamemode()), (string) GameModeIdMap::getInstance()->toId(GameMode::SURVIVAL()));
-		}while($gamemode < 0 or $gamemode > 3);
-		$config->set("gamemode", $gamemode);
+			$gamemodeInput = (int) $this->getInput($this->lang->translate(KnownTranslationFactory::default_gamemode()), (string) GameModeIdMap::getInstance()->toId(GameMode::SURVIVAL()));
```

Comment:
I'd rather just ditch the IDs and use GameMode::fromString() exclusively, so that the aliases used can be the same used by the /gamemode command. Less confusing for new users that way.

Reply:
I also mentioned this in my original post. I definitely agree that it would be better to just use the string version, but we would need to update language files to match.

Reply:
Would it be worth having `gamemode_info` in the language files accept arguments of all of the gamemode names?

Reply:
A single parameter which accepts `implode(", ", gameModeNames)` would be enough, I think.